tests/posix/sigaltstack.c

```diff
@@ -38,7 +38,7 @@ int main() {
 #elif defined(__aarch64__)
 	asm volatile ("mov sp, %0\n\tstp x0, x1, [sp, #-16]!" :: "r"(uint64_t{0}));
 #elif defined(__riscv) && __riscv_xlen == 64
-	asm volatile ("unimp"); // TODO
+	asm volatile ("li sp, 0\n\tsd zero, 0(sp)");
```

Comment:
If you use `\n\t` in inline assembly please also split the inline asm onto two lines.